routes/index.js

```diff
@@ -1,9 +1,57 @@
-var express = require('express');
+var express = require("express");
 var router = express.Router();
+const util = require("util");
+
+// Database connection
+let knex = require("knex")({
+    client: "pg",
+    connection: {
+        host: process.env.DB_HOST,
+        port: process.env.DB_PORT,
+        user: process.env.DB_USER,
+        password: process.env.DB_PASS,
+        database: process.env.DB_NAME
+    }
+});
 
 /* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+router.get("/", function(req, res, next) {
```

Comment:
typically prefer lambda `(args) => {body}` to actual function `function(args) { body }`